oak-search-elastic/pom.xml

```diff
@@ -187,6 +187,12 @@
       <version>${project.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.jackrabbit</groupId>
+      <artifactId>oak-jcr</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
```

Comment:
If we added this dep only for the test classes in oak-jcr, then we should limit the scope to test

Reply:
done